docs/UserGuide.md

```diff
@@ -190,15 +185,15 @@ Format: `exit`
 
 #### Saving the data
 
-Projact data are saved in your hard disk automatically after any command that changes the data so there is no need to save manually. Don't worry, the data you have entered in previously will be automatically loaded when you reopen Projact the next time.
+Projact's data is saved to your hard disk automatically after any changes in the data so there is no need to save manually. Don't worry, the data you have entered in previously will be automatically loaded when you reopen Projact the next time.
```

Comment:
"... after there are any changes in the data..."? Not sure about the sentence flow/grammar

Reply:
okie, thanks :D